seed/run.sh

```diff
@@ -1 +1,3 @@
-mongoimport --db colorschemes --collection repositories --jsonArray --drop --file $(pwd)/seed/data.json
+SCRIPT_DIR=${REMOTE_SEED_DIR:-$(cd $(dirname $0); pwd)}
```

Comment:
`$(pwd)` may be `/usr/local/bin` when we run `db` container by docker-compose. So I changed the seed directory:
- If `REMOTE_SEED_DIR` is set (in other words: run in db container), value of `REMOTE_SEED_DIR`
- Otherwise, the same directory as run.sh
  - This approach cannot be used in db container